Makefile

```diff
@@ -6,16 +6,17 @@ GOTOOLS = github.com/golang/lint/golint \
 # Temporary patch to avoid build failing because of the outdated documentation example
 # TODO: uncomment below commented lines and remove any line that uses $(NODOCS)
 NODOCS = $(shell go list ./... | grep -v /docs/)
+PKGS = $(shell go list ./... | egrep -v "\/docs\/|jmx")
 
-all: validate test
+all: lint test
 
 deps: tools
 #	@go get -v -d -t ./...
-	@go get -v -d -t $(NODOCS) #todo:remove
+	@go get -v -d -t $(NODOCS) #TODO: remove
 
 test: deps
-#	@gocov test ./... | gocov-xml > coverage.xml
-	@gocov test $(NODOCS) | gocov-xml > coverage.xml
+	@gocov test github.com/newrelic/infra-integrations-sdk/jmx > /dev/null # TODO: fix race for jmx package
+	@gocov test -race $(PKGS) | gocov-xml >> coverage.xml
```

Comment:
Appending output to `coverage.xml`?

Reply:
+1 to this question. Why appending?

Reply:
Yes, that'd be great, but we didn't know how to do it and it could block us for a while, we decided to keep `jmx` out of the coverage.

Feel free to update it.

Reply:
We were referring to the `gocov-xml >> coverage.xml`, with double `>>`. Won't this generate an invalid XML?